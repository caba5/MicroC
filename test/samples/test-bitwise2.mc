void main(){
    int bit1 = 5;
    int bit2 = 2;
    print(bit1 | bit2);
    print(bit1 & bit2);
    print(bit1 ^ bit2);
    print(2 << 2);
    print(8 >> 2);
    print(~bit2);
    print(bit1 | bit2 ^ (bit1 << 2) >> 8);
}
