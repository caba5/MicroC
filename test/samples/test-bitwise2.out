7
0
7
8
2
-3
7
